lib/cleanup_group.c

```diff
@@ -199,9 +201,11 @@ void cleanup_unlock_group (MAYBE_UNUSED void *arg)
  *
  * It should be registered after the gshadow file is successfully locked.
  */
-void cleanup_unlock_gshadow (MAYBE_UNUSED void *arg)
+void cleanup_unlock_gshadow (void *process_selinux)
 {
-	if (sgr_unlock (true) == 0) {
+	bool process = *((bool *)process_selinux);
```

Comment:
Please use a space between a cast and the expression it casts.  Using a consistent style makes it easier to grep for casts.

Alternatively, you could use a temporary pointer:

```c
void
cleanup_unlock_gshadow(void *process_selinux)
{
        const bool  *ps = process_selinux;
        bool        process = *ps;
```

which avoids the cast altogether.

Reply:
I prefer casting in a line. Updated.